tests/test.test.js

```diff
@@ -0,0 +1,97 @@
+const request = require("supertest");
+const { app } = require("../app");
+const { dataBase } = require("../routes/shorturlRoutes");
+
+jest.setTimeout(10000);
+
+let validUrlResponseBody, shortIdInDataBase;
+
+beforeAll(async (done) => {
+	await dataBase.updateSelf();
+	done();
+});
+
+describe("PUT Route to /api/shorturl", () => {
+	const validLongUrl = "https://www.w3schools.com/jsref/jsref_obj_date.asp";
+
+	it("Should put and return the new shortened url", async (done) => {
+		const requestBody = { longUrl: validLongUrl };
+		const response = await request(app).put(`/api/shorturl`).send(requestBody);
+
+		shortIdInDataBase = response.body.shortUrlId;
+		validUrlResponseBody = response.body;
+		console.log(response.body);
+		// Is the status code 200
+		expect(response.status).toBe(200);
+
+		// Is the response body structured like a url object
+		expect(response.body.long).toBeDefined();
+		expect(response.body.short).toBeDefined();
+		expect(response.body.shortUrlId).toBeDefined();
+		expect(response.body.clickCount).toBeGreaterThanOrEqual(0);
+		expect(response.body.creationDate).toBeGreaterThan(1000000000000);
+		done();
+	});
```

Comment:
Seems good. A note and a tip:
* Request method is supposed to be `POST`, isn't it?
* If you have access to your DB, fetch the checked entry and use it in your assertions (`expects()`) instead of `toBeDefined()`